Removing conda
Loading conda
Thu Feb 13 21:14:13 UTC 2025
SWOT_L2_HR_PIXC_020_035_229L_20240821T213732_20240821T213743_PIC0_01
(793654,)
type: normal
NHDPLUS_H_0107_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0108_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0109_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0110_HU4_GDB
flowlines read-in
exploded
INFO: Pandarallel will run on 64 workers.
INFO: Pandarallel will use Memory file system to transfer data between the main process and workers.
slurmstepd-umd-cscdr-cpu011: error: *** JOB 28915728 ON umd-cscdr-cpu011 CANCELLED AT 2025-02-13T21:14:36 ***
slurmstepd-umd-cscdr-cpu011: error: container_p_join: open failed for /var/tmp//umd-cscdr-cpu011/28915728/.ns: No such file or directory
slurmstepd-umd-cscdr-cpu011: error: container_g_join(28915728): No such file or directory
Traceback (most recent call last):
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/managers.py", line 813, in _callmethod
    conn = self._tls.connection
           ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'

During handling of the above exception, another exception occurred:

